configs/config.yaml

```
@@ -88,4 +88,21 @@ logging:
+    cdn_endpoint: ""  # Set via environment variable BUCKET_CDN_ENDPOINT  
+    access_key_id: ""  # Set via environment variable BUCKET_ACCESS_KEY_ID
+    access_key_secret: ""  # Set via environment variable BUCKET_ACCESS_KEY_SECRET
+    region: "blr1"  # Default region
+    bucket_name: "letraz-all-purpose"  # Default bucket name
+
+# Resume client configuration for screenshot generation
+resume:
+  client:
+    base_url: "http://localhost:3000"  # Default letraz-client URL (deprecated, use preview_url)
+    preview_url: "http://localhost:3000/admin/resumes"  # Full URL for resume preview endpoint
+    preview_token: ""  # Set via environment variable RESUME_PREVIEW_TOKEN
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)